cves/CVE-2009-2902.yml

```diff
@@ -43,15 +43,21 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: This vulnerability allows a user over a network to create a specially
+  crafted WAR file that could be deployed by an authorized user. The WAR could
```

Comment:
Consider defining what WAR is, just because this is supposed to be a general description